### ISSUE TITLE
[Support] Accessing annotations of a property for the ClassField

### ISSUE THREAD

**Issue by Airmanbzh** [goaop/framework#387]
Hello !

I want to annotate some properties to set a defaut value. I made an advice which work well to catch access to annotated properties but I can't access to these annotations once I'm there.

My annotation :
```php
namespace AZE\aop\annotation;

use Doctrine\Common\Annotations\Annotation;

/**
 * @Annotation
 *
 * @Target("ALL")
 */
class Config extends Annotation
{
    public $instance;
}
```

My annotated class :
```php
namespace data\config;

use AZE\aop\annotation\Config;

class Configuration
{
    /**
     * @Config("application")
     */
    public $application = "application_set";

    /**
     * @Config("my.custom.value")
     */
    public $value = "value_set";

    public $model = "aze";
}
```

```php
namespace AZE\aop\aspect;

use Go\Aop\Aspect;
use Go\Aop\Framework\ClassFieldAccess;
use Go\Lang\Annotation\Around;

class ConfigAspect implements Aspect
{
    /**
     * @Around("@access(AZE\aop\annotation\Config)")
     */
    public function getConfigValueAround(ClassFieldAccess $field)
    {
        var_dump($this);
        var_dump($field);
        return "around";
    }
}
```

I don't see any possibility to access current annotations of the property once I'm in my advice. Is it possible ?

Thanks :)

**Comment by TheCelavi** [comment]
Oh! Seams not, right @lisachenko ?

You can read it via used annotation reader, GoAOP uses Doctrine Annotation, AspectKernel is singleton, so you can use service locator to get annotation reader to read annotation from reflection property:

`AspectKernel::getInstance()->getContainer()->get('aspect.annotation.reader');`

But seams very strange that there is no more elegant way to do so, if Alexandar confirms, we can provide quick and easy fix for this.

**Comment by lisachenko** [comment]
HI, everyone :) I think that this feature wasn’t implemented because no one previously has asked for it.

Fix for that is straightforward. PR is welcome :)

**Comment by Airmanbzh** [comment]
Thanks a lot, to both, for your reactivity :)

**Comment by lisachenko** [comment]
Implemented in #388 by @TheCelavi, merged into master and will be released in 3.x version.
